### PR TITLE
Github action and fixing the script

### PR DESCRIPTION
**Pull request by walex999**
Fixing the script which was not working anymore
And creation of a github action to update the playground through a workflow_dispatch

Note: a gitlab secret should be created for the Stream and Horizon repositories -> API_TOKEN_GITLAB

### CODE REVIEW

**Review by milairhu — APPROVED** [review]

File: .github/workflows/bump.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+      - name: Grant rights
+        run: |
+          sudo chown -R $USER:$USER ${{ env.REPO_PATH }}
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          title: Update Playground
+          commit-message: Peter has Updated the playground
```
Comment (antoninguyot) [inline .github/workflows/bump.yml]: ???

File: .github/workflows/bump.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      - name: Display values
+        run: |
+          echo "ROOT_PATH: ${{ env.ROOT_PATH }}"
+          echo "REPO_PATH: ${{ env.REPO_PATH }}"
+
+      - name: Cloning Stream
+        if: ${{ inputs.stream-target-version }}
+        run: |
+          cd ${{ env.ROOT_PATH }}
+          git clone -b v${{ inputs.stream-target-version }} "https://milairhu:${{ secrets.API_TOKEN_GITLAB }}@gitlab.com/evertrust/stream.git"
```
Comment (antoninguyot) [inline .github/workflows/bump.yml]: The entire clone URL should be a variable

**Review by antoninguyot — CHANGES_REQUESTED** [review]
Si ca marche ~ les 2-3 commentaires, LGTM

File: .github/workflows/bump.yml (hunk 1 above)
Comment (walex999) [reply .github/workflows/bump.yml]: Fixed c61ac5a43d8e8e66c2118184e411a2707bae2210

File: .github/workflows/bump.yml (hunk 2 above)
Comment (walex999) [reply .github/workflows/bump.yml]: Fixed c61ac5a43d8e8e66c2118184e411a2707bae2210

**Review by ArcRiiad — APPROVED** [review]

**Pull request merged**